src/strings/en-gb.json

```diff
@@ -1457,6 +1457,8 @@
     "LabelEnableIP4Help": "Enables IPv4 functionality.",
     "LabelEnableIP4": "Enable IPv4:",
     "HeaderNetworking": "IP Protocols",
+    "EnablePlugin": "Enable Plugin",
+    "DisablePlugin": "Disable Plugin",
```

Comment:
Please note for future @BaronGreenback that `en-us` is the translation base, not `en-gb`.

Reply:
I corrected this manually in the cherry-pick by including the translations in both `en-us` and `en-gb`. It needs to still be corrected in `master` in a separate PR but I'm not waiting around for that.